guides/v2.2/cloud/project/project-conf-files_services-mysql.md

```diff
@@ -55,39 +55,42 @@ mysqldb:
             importer:
```

Comment:
I would also describe that in this case you need to change .magento.app.yaml
So for this example we should have next records in `relationships` section:
`relationships:
    database: "mysqldb:admin"
    databasereporter: "mysqldb:reporter"
    databaseimporter: "mysqldb:importer"`

Reply:
and we always should have connection with name `database` with admin rights. You cannot set `database: "mysqldb:reporter"`

Reply:
by default, we have `database: "mysql:mysql"` where `mysql` after colon - endpoint with admin permissions - the same as you have `admin in the example above

Reply:
Also maybe make sense to add link to page about master-slave DB and redis